docs/src/getting_started.md

```diff
@@ -11,7 +11,18 @@ using Pkg
 Pkg.add(ClimaLand)
 ```
 
-## Running a simple soil model
+## Steps to run a model
+
+To run a model, you will always need to do these steps:
+- Import dependencies. You need some libraries to run the model.
+- Chose Float32 or Float64.
+- Import parameters. You need some constants and parameters values to run the model.
+- Define your domains. You need a domain to solve PDEs. See the [tutorial to ClimaLand domains!](@ref "Domain Tutorial")
+- Set your simulation period: start date, end date, timestep.
+- Import the forcings (for example, air temperature, precipitations).
+- Set the initial conditions. PDEs are solved within a domain, advancing in time from initial conditions. (For example, initial soil moisture).
+
+## Example: running a simple soil model
```

Comment:
I think we might also include:
- "Set how output is saved. For some simulations (e.g., global land simulations), we use the default diagnostic output, corresponding to monthly averages of specific variables saved to netcdf files. For your use case, you may wish to change these defaults. See X for details."